YPP-0019.md

```diff
@@ -0,0 +1,228 @@
+  [ETH, '0x3bDb887Dc46ec0E964Df89fFE2980db0121f0fD0'],
+  [DAI, '0x4fE92119CDf873Cf8826F4E6EcfD4E578E3D44Dc'],
+  [USDC, '0x0d9A1A773be5a83eEbda23bf98efB8585C3ae4f4'],
+  [WBTC, '0x00De0AEFcd3069d88f85b4F18b144222eaAb92Af'],
+  [WSTETH, '0x5364d336c2d2391717bD366b29B6F351842D7F82'],
+  [LINK, '0xbDaBb91cDbDc252CBfF3A707819C5f7Ec2B92833'],
+  [ENS, '0x5AAfd8F0bfe3e1e6bAE781A6641096317D762969'],
+  [YVUSDC, '0x403ae7384E89b086Ea2935d5fAFed07465242B38'],
+  [UNI, '0x41567f6A109f5bdE283Eb5501F21e3A0bEcbB779'],
+])
+```
+
+## Proposal Config
+```
+export const governance: Map<string, string> = base_config.governance
+export const protocol: Map<string, string> = base_config.protocol
+export const assets: Map<string, string> = base_config.assets
+export const joins: Map<string, string> = base_config.joins
+export const newFYTokens: Map<string, string> = base_config.newFYTokens
+export const newPools: Map<string, string> = base_config.newPools
+export const newStrategies: Map<string, string> = base_config.newStrategies
+
+export const rateChiSources: Array<[string, string, string, string]> = [
+  [ETH, RATE, WAD.toString(), WAD.toString()],
+  [ETH, CHI, WAD.toString(), WAD.toString()],
+]
+
+// Assets that will be made into a base
+export const bases: Array<[string, string]> = [[ETH, base_config.joins.get(ETH) as string]]
+
+// Input data: baseId, ilkId, ratio (1000000 == 100%), line, dust, dec
+export const chainlinkDebtLimits: Array<[string, string, number, number, number, number]> = [
+  [ETH, DAI, 1400000, 1000000000, 1250000, 12],
+  [ETH, USDC, 1400000, 1000000000, 1250000, 12],
+  [ETH, ETH, 1000000, 2500000000, 0, 12], // Constant 1, no dust
+]
+
+export const newChiSources: Array<[string, string]> = [[ETH, '0x4ddc2d193948926d02f9b1fe9e1daa0718270ed5']]
+
+export const newRateSources: Array<[string, string]> = [[ETH, '0x4ddc2d193948926d02f9b1fe9e1daa0718270ed5']]
+
+// Input data: baseId, ilkId, oracle name, ratio (1000000 == 100%), inv(ratio), line, dust, dec
+export const newChainlinkLimits: Array<[string, string, number, number, number, number]> = [
+  [ETH, ETH, 1000000, 2500000000, 0, 12], // Constant 1, no dust
+  [ETH, DAI, 1500000, 250000000, 10000, 12],
+  [ETH, USDC, 1500000, 250000000, 10000, 12],
+  [ETH, WBTC, 1500000, 250000000, 10000, 12],
+  [ETH, LINK, 1500000, 250000000, 10000, 12],
+  [ETH, UNI, 1500000, 250000000, 10000, 12],
+]
+
+export const newCompositePaths: Array<[string, string, Array<string>]> = [[WSTETH, ETH, [STETH]]]
+
+// Input data: baseId, ilkId, oracle name, ratio (1000000 == 100%), inv(ratio), line, dust, dec
+export const newCompositeLimits: Array<[string, string, number, number, number, number]> = [
+  [ETH, WSTETH, 1000000, 250000000, 10000, 12],
+]
+
+// Input data: baseId, ilkId, oracle name, ratio (1000000 == 100%), inv(ratio), line, dust, dec
+export const newUniswapLimits: Array<[string, string, number, number, number, number]> = [
+  [ETH, ENS, 1500000, 250000000, 10000, 12],
+]
```

Comment:
Shouldn't there be an "oracle name" as per the comment?

Reply:
Fixed natspec, no need for an oracle name.